enterprise-ws-spec/src/main/asciidoc/Handlers.adoc

```diff
@@ -125,146 +110,95 @@ Common scenarios are logging, metering, and accounting.
+* The handler instance is initialized by invoking the lifecycle method annotated with 
+jakarta.annotation.PostConstruct (defined by Jakarta Annotations specification).
+Once the handler instance is created and initialized it is placed into the Ready state. 
+While in the Ready state the Jakarta XML Web Services runtime may invoke other handler 
+methods as required.
 
-The init and destroy methods of the Handler interface allows the
-container to notify a Handler instance of impending changes in its
-state. The Handler may use the notification to prepare its internal
-state for the transition. The container is required to call the init and
-destroy methods as described below.
-
-The container must call the init method before it can start dispatching
-requests to the handleRequest(), handleResponse(), and handleFault()
-methods of the Handler. The Handler may use the container notification
-to ready its internal state for receiving requests.
-
-The container must notify the Handler of its intent to remove the
-instance from the container’s working set by calling the destroy method.
-A container must not call the destroy method while a request is being
-processed by the Handler instance. The container must not dispatch
-additional requests to the Handler interface methods after the destroy
-method is called.
-
-As defined by JAX-RPC, a RuntimeException (other than
-SOAPFaultException) thrown from any method of the Handler results in the
-destroy method being invoked and transition to the “Does Not Exist”
-state.
+* The lifecycle of a handler instance ends when the container stops using the handler 
+for processing inbound or outbound messages and invokes the lifecycle method annotated
+with jakarta.annotation.PreDestroy.
 
-Pooling of Handler instances is allowed, but is not required. If Handler
-instances are pooled, they must be pooled by Port component. This is
-because Handlers may retain non-client specific state across method
-calls that are specific to the Port component. For instance, a Handler
-may initialize internal data members with Port component specific
-environment values. These values may not be consistent when a single
-Handler type is associated with multiple Port components. Any pooled
-instance of a Port component’s Handler in a Method Ready state may be
-used to service the handleRequest(), handleResponse(), and handleFault()
-methods. It is not required that the same Handler instance service both
-the handleRequest() and handleResponse() or handleFault() method
-invocations of any given request.
-
-===== Handler Life Cycle with JAX-WS
-
-With JAX-WS the Handler lifecycle is similar to JAX-RPC illustration
-shown in the previous section with the following differences:
-
-* The init() method is now replaced by a method carrying a
-javax.annotation.PostConstruct annotation (defined by JSR-250
-specification, Section 2.4). This method is only called after all
-injections requested by the Handler are completed.
```

Comment:
"This method is only called after all injections requested by the Handler are completed." sentence needs to be kept; in current wording it is unclear when exactly injection is supposed to happen - before or after a method annotated with PostConstruct?

Reply:
I see this is defined few paragraphs later... Anyway is there a real need to remove it from this place?

Reply:
No, it was result of removing JAX-RPC specific information. I will add it back in its original place.